integration/src/test/java/org/mule/test/integration/transaction/TransactionsWithRoutersTestCase.java

```diff
@@ -212,17 +212,20 @@ public void flowRefToFlowWithErrorAndPropagate() throws Exception {
   }
 
   @Test
-  @Ignore("MULE-17023")
-  public void nestedTriesWithOnErrorPropagatesAndContinue() throws Exception {
+  public void nestedTriesWithOnErrorPropagates() throws Exception {
     runsInSameThread("nestedTriesWithOnErrorPropagate", TX_MESSAGE, TX_MESSAGE, OTHER_TX_MESSAGE, OTHER_TX_MESSAGE);
   }
 
   @Test
-  @Ignore("MULE-17023")
   public void innerTryWithOnErrorPropagate() throws Exception {
     runsInSameThread("tryWithInnerTryWithOnErrorPropagate", TX_MESSAGE, TX_MESSAGE, OTHER_TX_MESSAGE, OTHER_TX_MESSAGE);
   }
 
+  @Test
+  public void nestedTriesWithOnErrorContinue() throws Exception {
+    runsInSameThread("nestedTriesWithOnErrorContinue", TX_MESSAGE, TX_MESSAGE, OTHER_TX_MESSAGE, OTHER_TX_MESSAGE);
```

Comment:
is this sameThread assertion just to check there was a tx? If so, check for an active tx also

Reply:
Yes, the function name is a bit deprecated, since it really checks that tx is active, threads are the same and paylaods are the expected ones